AppAuth-JS/built/token_request.d.ts

```diff
@@ -0,0 +1,35 @@
+import { StringMap } from './types';
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork